src/stream_wrap.cc

```diff
@@ -65,7 +65,7 @@ void LibuvStreamWrap::Initialize(Local<Object> target,
   Local<FunctionTemplate> sw =
       FunctionTemplate::New(env->isolate(), is_construct_call_callback);
   sw->InstanceTemplate()->SetInternalFieldCount(
-      StreamReq::kStreamReqField + 1 + 3);
+      StreamReq::kInternalFieldCount + 3);
```

Comment:
As mentioned above, this looks highly dubious. :-)

Reply:
I'm still not clear on where the other three fields are coming from or where they are used.

Reply:
I think @mcollina was thinking that the JS fields set below on the instance template were counted as internal fields when he added them. They don’t, and it’s safe (and correct) to remove the `+ 3`.

Reply:
Yes, I thought that.

Reply:
Should we backport that change seperately? Adding that +3 is an issue in production?